core/templates/dev/head/profile/Preferences.js

```diff
@@ -35,6 +36,43 @@ oppia.controller('Preferences', ['$scope', '$http', '$rootScope', '$modal', '$ti
     _saveDataItem('user_bio', userBio);
   };
 
+  $scope.subjectInterestsChangedAtLeastOnce = false;
+  $scope.subjectInterestsWarningText = null;
+  $scope.TAG_REGEX_STRING = '^[a-z ]+$';
+
+  $scope.updateSubjectInterestsWarning = function(subjectInterests) {
+    var TAG_REGEX = new RegExp($scope.TAG_REGEX_STRING);
+
+    if (subjectInterests instanceof Array) {
+      for (var i = 0; i < subjectInterests.length; i++) {
+        if (typeof subjectInterests[i] === 'string') {
+          if (!TAG_REGEX.test(subjectInterests[i])) {
+            $scope.subjectInterestsWarningText = (
+              'Subject interests can only have alphabetic characters.');
```

Comment:
Subject interests should use only lowercase letters.

(Trying to convey the notion of lowercase.)